zebra-consensus/src/checkpoint.rs

```diff
@@ -132,6 +135,13 @@ where
 
     /// The current progress of this verifier.
     verifier_progress: Progress<block::Height>,
+
+    /// A channel to receive requests to reset the verifier,
+    /// receiving the tip of the state.
+    reset_receiver: Receiver<Option<(block::Height, block::Hash)>>,
+    /// A channel to send requests to reset the verifier,
+    /// passing the tip of the state.
+    reset_sender: Sender<Option<(block::Height, block::Hash)>>,
```

Comment:
Readability: I try to qualify channel names with the channel type, because `rust-analyzer` sometimes doesn't show the qualified type.

```suggestion
    /// A channel to receive requests to reset the verifier,
    /// receiving the tip of the state.
    reset_receiver: mpsc::Receiver<Option<(block::Height, block::Hash)>>,
    /// A channel to send requests to reset the verifier,
    /// passing the tip of the state.
    reset_sender: mpsc::Sender<Option<(block::Height, block::Hash)>>,
```

Reply:
Good suggestion, changed in e3d1907b